tests/components/onvif/test_config_flow.py

```diff
@@ -379,6 +380,7 @@ async def test_flow_manual_entry(hass):
                 config_flow.CONF_NAME: NAME,
                 config_flow.CONF_HOST: HOST,
                 config_flow.CONF_PORT: PORT,
+                config_flow.CONF_FFMPEG_SNAPSHOT: FFMPEG_SNAPSHOT,
```

Comment:
To be clear, are you changing some existing tests to use this new configuration, rather than write completely new tests? If so are you sure this doesn't invalidate the original tests intent?

Reply:
Took me a minute to reply, sorry for that! My reading of the unir test in question is that configurations collected during the flow are correctly represented. This PR is adding a configuration option to the flow, I believe it should be covered by the existing testing harness. IE: if the configuration flow is somehow changed not to collect any of the parameters, the test would fail. The presence of the additional configuration option would not fail an otherwise successful test, nor - worst case scenario - make a failing test pass.

  That said: I can create a separate unit test just for this option, but I feel it could be redundant - on the other hand I'm new in this codebase, so I could be missing something! :)

Reply:
Not marking this as resolved, will wait for a response.